register_forte_options.py

```diff
@@ -158,6 +158,15 @@ def register_embedding_options(forte_options):
     forte_options.add_int(
         "EMBEDDING_ADJUST_B_UOCC", 0,
         "Adjust number of virtual orbitals between A and B, +: move to B, -: move to A")
+    forte_options.add_str(
+        "EMBEDDING_VIRTUAL_SPACE", "DIRECT", 
+        "Vitual space scheme, DIRECT, PAO or IAO.")
```

Comment:
When there is only a finite number of valid options, it is better to use the other versions of `forte_options.add_str` that allows to specify the allowed values. Basically, just add `['DIRECT', 'PAO', 'IAO'] after ` "DIRECT",`. Also, what does DIRECT stand for? It sounds very generic, what about DMET or ASET?

Reply:
I will use ASET instead.